Makefile

```diff
@@ -1,18 +1,13 @@
 SRC := .
 
 env:
-	mkvirtualenv --python=$(which python3.8) up42-py
-	workon up42-py
+	poetry shell
 
 install:
 	pip install up42-py
 
 install[dev]:
```

Comment:
The following steps need to change as well:
1. install needs to change to use poetry
2. pytest - to switch to poetry
3. e2e - to switch to poetry. Do we actually need this as a separate step?
4. serve - better to switch to poetry to make sure that it runs with the specified dependencies
5. gh-pages - poetry
6. package + upload - twine is not needed anymore, switch to poetry